lib/ZumoOta/src/BootloaderCom.h

```diff
@@ -99,7 +113,22 @@ class CmdProvider {
      *@param rsp Reference to a pointer to the next response.
      *@return True if there is a next command and response pair, otherwise false.
      */
-    virtual bool next(const CommandInfo *& cmd,  const ResponseInfo *& rsp) = 0;
+    virtual bool next( const CommandInfo *& cmd, const ResponseInfo *& rsp) = 0;
+
+    /**
+     *@brief Read 128 bytes from the file system.
+     */
+    static uint8_t* m_buffer;

```

Comment:
why a pointer ?

Reply:
It will be take care by the next commit.